src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/appointments/AppointmentsResource.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.prisonertonomisupdate.appointments
+
+import io.swagger.v3.oas.annotations.Hidden
+import org.springframework.http.HttpStatus
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class AppointmentsResource(private val appointmentsService: AppointmentsService) {
+
+  /**
+   * For dev environment only - delete all appointments, for when activities environment is reset
+   */
```

Comment:
Should we feature switch to allow in dev only?

Reply:
definitely - will add